### PR TITLE
Add get_first_available_zivid_device function

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This function prevents open_framegrabber from connecting to the TeliCamTL64 producer of the internal 2D camera. Without this function, Halcon connects to the first suitable device it finds in the GENICAM_GENTL_64_PATH. With this new function, the order in the path becomes irrelevant.

### CODE REVIEW

File: source/Camera/Basic/Capture.hdev [hunk 1]
```diff
@@ -75,4 +79,30 @@
...
+<l>endfor</l>
+<l>return ()</l>
+</body>
+<docu id="get_first_available_zivid_device">
+<parameters>
+<parameter id="Device"/>
+<parameter id="ValueList"/>
+</parameters>
+</docu>
+</procedure>
```
Comment (trym-b) [inline source/Camera/Basic/Capture.hdev]: Could you somehow move this function a shared file so all samples can use it?

Comment (SatjaSivcev) [reply source/Camera/Basic/Capture.hdev]: Good idea. I have no idea, I will try to figure out.

Comment (SatjaSivcev) [reply source/Camera/Basic/Capture.hdev]: Done!

**Comment by SatjaSivcev** [comment]
Few things make me as proud as figuring out something new in Halcon. Ready for review.

File: source/Camera/Basic/Capture.hdev (hunk 1 above)
Comment (trym-b) [reply source/Camera/Basic/Capture.hdev]: Looks good

**Comment by SatjaSivcev** [comment]
@torbsorb do you approve the proposed folder structure? If so, I will merge.

**Pull request merged**